doc/user-guide/template.md

```diff
@@ -42,6 +42,11 @@ Let's load some data and connect to the in-memory DuckDB instance:
 
 ```{code-cell} ipython3
 %load_ext sql
+%sql duckdb://
+%config SqlMagic.displaylimit = 3
```

Comment:
We need to add `displaylimit` [here](https://jupysql.ploomber.io/en/latest/api/configuration.html#) also

Reply:
good catch. added https://github.com/ploomber/jupysql/pull/770/commits/8d07db41ba3e4a3b79117b9a59e48681732a1ae0